.github/workflows/tutorials.yml

```diff
@@ -0,0 +1,38 @@
+name: Jupyter notebooks
+
+on:
+  # Trigger the workflow on push or pull request,
+  # but only for the master branch
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  test:
+    name: Test Jupyter notebooks on ${{ matrix.python-version }} and ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
```

Comment:
matrix does all combinations, which in this case is a bit much as probably only three cases are needed here. Choose one python version for each OS and one run per OS

Reply:
Would you be able to recommend which platforms would be preferred for testing these notebooks i.e. is it necessary to test on both `macOS` and `ubuntu` against both `clang` and `gcc` or perhaps is it only required to check `clang` and `gcc` on a single OS.

Additionally, is it overkill to test against multiple compiler versions, e.g `gcc-7, 8, 9 ..` etc

Reply:
`(macos, gcc)` and `(ubuntu, gcc)` should be enough.

Reply:
Also, multiple gcc version already tested in pytest, so a single one should be fine here, so basically should need only two builds.

Reply:
Resolved in [`651b253`](https://github.com/devitocodes/devito/pull/1069/commits/651b253d24e74d8a7bc8808a04d63a1887726c07)